pyproject.toml

```diff
@@ -6,8 +6,9 @@ requires = [
     "setuptools>=34.4",
 
     # Plugins
-    "setuptools_scm[toml]>=3.5",
-    "setuptools_scm_git_archive>=1.1",
+    "setuptools-scm >= 7.0.0,
+    "setuptools-scm[toml] >= 3.5, < 7.0.0,
+    "setuptools-scm-git-archive >= 1.1,
```

Comment:
This also contains syntax same errors. And also, it effectively adds conflicting dependencies — `setuptools-scm` can't be both below and above v7 simultaneously. The "below" version was only relevant earlier when we supported Python < 3.7. Now that we don't, having it is pointless. The `git-archive` plugin mustn't be paired with `setuptools-scm > 7`, it's not needed in such case and probably creates problems.
```suggestion
```